projects/nblocks-ionic/src/lib/auth/auth.service.ts

```diff
@@ -19,19 +19,32 @@ export class AuthService {
     tenantUsers: "/auth-proxy/tenantUsers",
     currentUser: "/auth/currentUser",
     password: "/auth-proxy/password",
-    user: "/auth-proxy/user"
+    user: "/auth-proxy/user",
+    socialLogin: "/auth-proxy/social-login"
   }
 
   private BASE_URL: string;
+  private NBLOCKS_APP_ID: string;
 
   private readonly _currentUserSource = new BehaviorSubject<CurrentUser>(new CurrentUser(false));
   readonly currentUser$ = this._currentUserSource.asObservable();
 
+  readonly GOOGLE_BUTTON_ENABLED: boolean = false;
```

Comment:
it was my initial approach, but as we already inject nblocksLibService in AuthService, I did it in this way to avoid double injection.
no problem. will do revert